demo/compose/compose.yaml

```diff
@@ -25,5 +11,15 @@ services:
       dockerfile: demo/compose/adapter/Dockerfile
       args:
         - LOG_FILE=okuma.txt
-
-    
+  agent:
+    image: "mtconnect/agent:latest"
+    volumes:
+      - ./agent/agent.cfg:/mtconnect/config/agent.cfg
+      - ../agent/Devices.xml:/mtconnect/config/Devices.xml
```

Comment:
Do we need to bind-mount `agent.cfg` and `Devices.xml` when we copy them into the image at build time? :thinking:

Reply:
These are different files, not the ones that are in the image. This is not using the demo image, it's the base image.